### DISCUSSION TITLE
Exporting tours

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1045] [General]
Hi
Can I export all tours in one operation?
Thanks

**Comment by FJBDev** [comment]
Yes, you can export all of them by selecting all the tours in the TourBookView for example and use the [export function](https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/export-tour)

**Reply by gringojack** [reply]
Thank you for info. But is it possible to export them with the name I gave them?

**Reply by FJBDev** [reply]
> But is it possible to export them with the name I gave them?

I don't understand your question, are you talking about the file name ? activity name ?
Also, which export format are you using ? TCX, GPX, MT ?

**Reply by gringojack** [reply]
Hi
I understand that selected file may be exported along with their own "date default" name. In MTB, I give them an other name stipulating "bike type, place, date". These names I would like them to keep. Is it possible?

**Reply by FJBDev** [reply]
> Hi I understand that selected file may be exported along with their own "date default" name. In MTB, I give them an other name stipulating "bike type, place, date". These names I would like them to keep. Is it possible?

Yes, you can enter your own filename as below:

![image](https://user-images.githubusercontent.com/25235665/228899025-95013bb7-be4d-41c9-ac26-777a242d8944.png)

**Reply by gringojack** [reply]
More precisely, I wish to export, all together  the files I selected with their own names (using the way explained above):

![MTBFileNames](https://user-images.githubusercontent.com/4082252/228934041-1d2f5ced-9397-4970-89d5-8cecd2bd66b6.jpg)

**Reply by FJBDev** [reply]
> More precisely, I wish to export, all together the files I selected with their own names (using the way explained above):
> 
Then I think in this case, if you want a filename for each activity, you will have to export each activity separately.

**Reply by gringojack** [reply]
Thank you
